src/v2/Apps/Partner/Components/NavigationTabs/index.tsx

```diff
@@ -1,15 +1,36 @@
+import { breakpoints } from "@artsy/palette"
 import React from "react"
 import { createFragmentContainer, graphql } from "react-relay"
+import styled, { css } from "styled-components"
+import { HorizontalPadding } from "v2/Apps/Components/HorizontalPadding"
+import { FullBleed } from "v2/Components/FullBleed"
 import { RouteTab, RouteTabs } from "v2/Components/RouteTabs"
+import { BaseContainer, StickyContainer } from "v2/Components/StickyContainer"
 import { NavigationTabs_partner } from "v2/__generated__/NavigationTabs_partner.graphql"
 
 interface NavigationTabsProps {
   partner: NavigationTabs_partner
 }
 
+export const Container = styled(BaseContainer)`
+  ${({ stuck }) =>
+    stuck
+      ? css`
+          & > div {
+            box-shadow: 0px 0px 1px rgba(0, 0, 0, 0.08),
+              0px 2px 8px rgba(0, 0, 0, 0.12);
+          }
+        `
+      : css`
+          & > div {
+            box-shadow: none;
+          }
+        `};
+`
+
 export const NavigationTabs: React.FC<NavigationTabsProps> = ({ partner }) => {
   const renderTabs = () => {
-    const { slug } = partner
+    const { slug, locations, articles } = partner
 
     const route = (path?: string) => `/partner2/${slug}${path ? path : ""}`
```

Comment:
We got one problem with using slug field here, when you go to https://staging.artsy.net/partner2/gagosian-gallery overview tab is no highlighted. It happens because of partner slug and partner id differences. Previous version uses profile id field: [link](https://github.com/artsy/force/blob/a1c2a40d4c06224eedd8e35beb0ce3acc44f398f/src/desktop/apps/partner2/templates/tablist.jade#L12). 
So, maybe I should use an another field instead of slug?

Reply:
For this page, we'll want to use the partner slug. Where are we doing the comparison in order to highlight the tab? Perhaps we need to update that.

Old pages/routes used the profile slug, which could potentially be different.

Before we ship this project, we'll need to handle redirecting from `/:profileSlug` -> `/partner/:partnerSlug`, but that should be a separate ticket.

Reply:
@sweir27  The comparision doing here: https://github.com/artsy/force/blob/10db215f80bcdbf1936e40b34f2f2f21b448e9a1/src/v2/Artsy/Router/useRouter.tsx#L12-L21
The function takes our [routes ](https://github.com/artsy/force/blob/10db215f80bcdbf1936e40b34f2f2f21b448e9a1/src/v2/Apps/Partner/Components/NavigationTabs/index.tsx#L38-L71) and compare with current url.
What do you think if we add a redirect from `/partner2/:partnerId` to `/partner2/:slug` if `partnerId ` is not equal to `slug`?

For example: `/partner2/gagosian-gallery` -> `/partner2/gagosian`

Reply:
I'm not clear if we're hitting issues because of a partner's `internalID` vs. a partner's `slug`, or whether it's because we're conflating a _profile_'s `slug` with the _partner_'s `slug`.

@dzucconi does any of this look familiar from what you dealt with w/ fair routing?

Reply:
This just looks like some partners have multiple slugs

![](https://static.damonzucconi.com/_capture/T7Ix0ntH.png)

I'd expect this to be pretty rare? We could just ensure we redirect to the canonical one if another one gets hit

Reply:
Yeah-- @kiryl-zubarau this also feels like it could be a separate ticket

Reply:
Thanks! Okay, we will create an additional ticket in JIRA for this.